crates/subspace-transaction-pool/src/lib.rs

```diff
@@ -417,18 +417,12 @@ where
         task_manager,
         tx_pre_validator,
     ));
-    let pool = Arc::new(BasicPoolWrapper::with_revalidation_type(
+
+    Arc::new(BasicPoolWrapper::with_revalidation_type(
         config,
         pool_api,
         prometheus,
         task_manager.spawn_essential_handle(),
         client.clone(),
-    ));
-
-    // make transaction pool available for off-chain runtime calls.
-    client
-        .execution_extensions()
-        .register_transaction_pool_factory(OffchainTransactionPoolFactory::new(&pool));
```

Comment:
The test failed due to tx pool is probably because these are deleted and the equivalents are not added.